.testing/chimp.js

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+"use strict";
+
+var path = require('path');
+var extend = require('util')._extend;
+var baseDir = path.resolve(__dirname, '../');
+var srcDir = path.resolve(baseDir);
+var source = require(srcDir + '/node_modules/shell-source');
+var processes = require('./processes.js');
+var isCi = process.argv[2] === '--ci';
+
+var startTestApp = function(onStarted, options) {
+  return processes.start({
+    name: 'Test App',
+    command: 'meteor --port=3100',
+    waitForMessage: 'App running at: http://localhost:3100',
+    options: {
+      cwd: srcDir,
+      env: extend(process.env, options)
+    }
+  }, function() {
+    console.log("Test app is running …");
+    onStarted();
+  });
+};
+
+var startChimpWatch = function() {
+  processes.start({
+    name: 'Chimp Watch',
+    command: 'chimp --ddp=http://localhost:3100 --watch --path=tests --mocha --chai --browser=chrome',
+    options: { cwd: baseDir }
+  });
+};
+
+var startChimpCi = function() {
+  var command = 'chimp --ddp=http://localhost:3100 --path=tests --browser=chrome --mocha --chai';
+  processes.start({
+    name: 'Chimp CI',
+    command: command,
+    options: { cwd: baseDir }
+  });
+};
+
+if (isCi) {
+  // CI mode -> run once
+  if (process.env.CIRCLECI) {
+    startTestApp(startChimpCi);
+  } else {
+    // Use a different db for local ci testing to avoid nuking of the dev db
+    startTestApp(startChimpCi, {
+      MONGO_URL: 'mongodb://localhost:3001/chimp_db'
+    });
+  }
+} else {
+  // DEV mode -> watch
+  startTestApp(startChimpWatch, {
+    MONGO_URL: 'mongodb://localhost:3001/chimp_db'
```

Comment:
This line is causing errors for me when I run `./tests/acceptance_run` or `./tests/acceptance_watch`. If I comment it out, the tests run fine.

Any ideas?

```
W20160802-13:09:00.524(-7)? (STDERR)
W20160802-13:09:00.525(-7)? (STDERR) /Users/noah/.meteor/packages/meteor-tool/.1.3.4_4.l7vwzg++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:313
W20160802-13:09:00.525(-7)? (STDERR) 						throw(ex);
W20160802-13:09:00.525(-7)? (STDERR) 						      ^
W20160802-13:09:00.676(-7)? (STDERR) Error: failed to connect to [localhost:3001]
W20160802-13:09:00.676(-7)? (STDERR)     at Object.Future.wait (/Users/noah/.meteor/packages/meteor-tool/.1.3.4_4.l7vwzg++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:449:15)
W20160802-13:09:00.676(-7)? (STDERR)     at new MongoConnection (packages/mongo/mongo_driver.js:213:27)
W20160802-13:09:00.677(-7)? (STDERR)     at new MongoInternals.RemoteCollectionDriver (packages/mongo/remote_collection_driver.js:4:16)
W20160802-13:09:00.677(-7)? (STDERR)     at Object.<anonymous> (packages/mongo/remote_collection_driver.js:38:10)
W20160802-13:09:00.678(-7)? (STDERR)     at Object.defaultRemoteCollectionDriver (packages/underscore/underscore.js:750:1)
W20160802-13:09:00.678(-7)? (STDERR)     at new Mongo.Collection (packages/mongo/collection.js:103:40)
W20160802-13:09:00.678(-7)? (STDERR)     at AccountsServer.AccountsCommon (packages/accounts-base/accounts_common.js:23:18)
W20160802-13:09:00.678(-7)? (STDERR)     at new AccountsServer (packages/accounts-base/accounts_server.js:18:5)
W20160802-13:09:00.678(-7)? (STDERR)     at meteorInstall.node_modules.meteor.accounts-base.server_main.js (packages/accounts-base/server_main.js:9:12)
W20160802-13:09:00.679(-7)? (STDERR)     at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160802-13:09:00.679(-7)? (STDERR)     - - - - -
W20160802-13:09:00.679(-7)? (STDERR)     at [object Object].<anonymous> (/Users/noah/.meteor/packages/npm-mongo/.1.4.45.1n9eimd++os+web.browser+web.cordova/npm/node_modules/mongodb/lib/mongodb/connection/server.js:556:74)
W20160802-13:09:00.679(-7)? (STDERR)     at [object Object].emit (events.js:106:17)
W20160802-13:09:00.679(-7)? (STDERR)     at [object Object].<anonymous> (/Users/noah/.meteor/packages/npm-mongo/.1.4.45.1n9eimd++os+web.browser+web.cordova/npm/node_modules/mongodb/lib/mongodb/connection/connection_pool.js:156:15)
W20160802-13:09:00.680(-7)? (STDERR)     at [object Object].emit (events.js:98:17)
W20160802-13:09:00.680(-7)? (STDERR)     at Socket.<anonymous> (/Users/noah/.meteor/packages/npm-mongo/.1.4.45.1n9eimd++os+web.browser+web.cordova/npm/node_modules/mongodb/lib/mongodb/connection/connection.js:534:10)
W20160802-13:09:00.680(-7)? (STDERR)     at Socket.emit (events.js:95:17)
W20160802-13:09:00.680(-7)? (STDERR)     at net.js:441:14
W20160802-13:09:00.681(-7)? (STDERR)     at process._tickCallback (node.js:458:13)
```